external_parser/joiners/multistep_example_joiner.h

```diff
@@ -23,6 +24,36 @@
 
 namespace v2 = reinforcement_learning::messages::flatbuff::v2;
 
+enum multistep_reward_funtion_type {
```

Comment:
reward stuff could be moved to `reward.h` or to a `reward_multistep.h` that imports `reward.h` so that it can access `outcome` struct

Reply:
Not sure if we need to have access to outcome here. If original reward function is the function of outcome structs, then multistep_reward is function over graph (just list in this simple prototype) with results of reward function in the vertices. It could change though, since this code is more prototypish than the rest of external_parser. That was the reason to:
- make it as function of vector<float>
- keep it inside multistep_joiner for now.